### DISCUSSION TITLE
Can't write filtered data into another table

### DISCUSSION THREAD

**Discussion by SBGE1904** [Budibase/budibase#12831] [Help] [answered]
Hi Community, since I'm a beginner with Budibase it might be a super simple question and answer. 
Setup: I connected a Google Sheet with two Pages "Mitglieder" and "Eingabewerte". The simple thing I would like to do is to create a form in which the user can search and filter a Row from "Mitglieder" and write this exact row with all values into the table "Eingabewerte". I came across the filter-options in an Youtube tutorial and this works perfectly fine. But when it comes to save the filtered and selected row to the other table, I sometimes get an error "row does not exist" and sometimes the writing works. I could not find out when exactly which case takes place but I struggle to fix this issue. Is there a mistake I have made or something Im missing?
Hope that the screenshots provided help for understanding the issue.

This is my setup:
![grafik](https://github.com/Budibase/budibase/assets/157366441/dedba990-1f5d-4eb6-b814-5dc1e74f9475)
Filtering works fine:
![grafik](https://github.com/Budibase/budibase/assets/157366441/4308d456-2276-411d-8e8b-39d489e8b6bd)
When I press the button for saving the row, this error takes place:
![grafik](https://github.com/Budibase/budibase/assets/157366441/5bc205bb-c1a5-4ba0-ba38-3ead4cb15c65)
Setup of the save button:
![grafik](https://github.com/Budibase/budibase/assets/157366441/f88cc214-5282-48ae-a7cb-36ebab42f64e)
Setup of Mitgliederdaten:
![grafik](https://github.com/Budibase/budibase/assets/157366441/90129ff3-1715-43a3-bfe0-46a6d3f6fb47)
Setup of the data source Mitglieder
![grafik](https://github.com/Budibase/budibase/assets/157366441/030d18d4-7edd-43c7-8a2e-e9cc49d9c903)

**Comment by melohagan** [comment, marked as answer]
Hey @SBGE1904 Welcome! 👋 

The likely reason you are seeing this error is because the destination table does not match the source table. 

When you do a **Save Row** action with a table source, it will automatically take the **_id** from the source table (the unique row identifier) as it assumes by default you are updating an existing table row.

In your case, because you want to save an existing table row to a *new destination* table,  then make sure that the datasource is set to **None**.

![Screenshot 2024-01-22 at 14 21 23](https://github.com/Budibase/budibase/assets/101575380/22802a98-0a66-481b-9089-b9d80b8772fb)

This will tell Budibase to create a brand new row ID, and not to do an update 👍 

With that said, because you want to copy columns over from the other table, you will need to `Add Column` for each column and then use bindings to grab the value, e.g.
![Screenshot 2024-01-22 at 14 24 58](https://github.com/Budibase/budibase/assets/101575380/919dc8de-ac46-4224-8db4-beb5534e2956)

**Comment by SBGE1904** [comment]
Life can be sooo easy! :) Thank you so much for the quick help, it worked !
